src/lib.rs

```diff
@@ -45,6 +53,11 @@ include!(concat!(env!("OUT_DIR"), "/langs.rs"));
 ///
 /// This uses the default [bounds](Lang::bounds) for the language.
 ///
+/// # Panics
```

Comment:
I put the "Panics" section before the "Examples" section because that is how it is done in the standard library, but I can still move it if you prefer.

Reply:
Okay, then it's fine.